MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,10 +1,100 @@
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
+final class MovieQuizViewController: UIViewController, AlertPresenterDelegate, MovieQuizViewControllerProtocol {
+
     // MARK: - Lifecycle
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var textLabel: UILabel!
+    @IBOutlet weak private var counterLabel: UILabel!
+    @IBOutlet weak private var yesButton: UIButton!
+    @IBOutlet weak private var noButton: UIButton!
+
+    var alertPresenter: AlertPresenter?
```

Comment:
private?

Reply:
Я так пробовал сделать, возникает ошибка: Property 'alertPresenter' must be declared internal because it matches a requirement in internal protocol 'MovieQuizViewControllerProtocol'
Как я понял, чтобы можно было сделать эту переменную приватной, нужно вынести AlertPresenter в MovieQuizPresenter.